code/modules/projectiles/guns/projectile/automatic/idaho.dm

```diff
@@ -1,35 +1,38 @@
 /obj/item/weapon/gun/projectile/automatic/idaho
-	name = "FS SMG .35 Auto \"Idaho\""
-	desc = "The Idaho is a cheap self-defence weapon, mass-produced by \"Frozen Star\" for paramilitary and private use. Uses 30 Auto rounds."
+	name = "FS SMG .40 Magnum \"Idaho\""
+	desc = "The Idaho is an experimental SMG made by \"Frozen Star\" for paramilitary and private use. \
```

Comment:
"An experimental Submachine gun made by \"Frozen Star\", for Paramilitary and private security use. Rifled to take a larger caliber than a typical submachine gun, it boasts a greater impact, but suffers from poor recoil control as a result. Uses .40 Magnum rounds."

Reply:
Good one.